### PR TITLE
fix: updated en-US translation

### PR DESCRIPTION
**Pull request by gl14916**
Refined grammar and syntax of the app's English translation

### CODE REVIEW

File: src/shared/locales/en-US/task.js [hunk 1]
```diff
@@ -28,61 +28,61 @@ export default {
...
+  'resume-all-task-fail': 'Failed to resume all tasks',
   'clear-recent-tasks': 'Clear Recent Tasks',
   'purge-record': 'Purge Task Record',
-  'purge-record-success': 'Purge task record success',
-  'purge-record-fail': 'Purge task record fail',
+  'purge-record-success': 'Successfully purged task records',
+  'purge-record-fail': 'Failed to purge task records',
   'refresh-list': 'Refresh Task List',
-  'no-task': 'There are no current downloads',
+  'no-task': 'No active task',
```
Comment (agalwood) [inline src/shared/locales/en-US/task.js]: The `no-task` here is not just for the active task, the original text I borrowed from Folx Go+.

I suggest: `There are no current tasks.`

File: src/shared/locales/en-US/preferences.js [hunk 2]
```diff
@@ -6,17 +6,17 @@ export default {
...
   'startup': 'Startup',
   'open-at-login': 'Open at login',
-  'keep-window-state': 'Restore size and position of the window',
-  'auto-resume-all': 'Auto resume all unfinished tasks',
-  'default-dir': 'Default Dir',
-  'mas-default-dir-tips': 'Due to the sandbox permissions restrictions of the App Store, the default download dir is recommended to be set to Downloads directory',
-  'task-manage': 'Task Manage',
+  'keep-window-state': 'Keep size and position of the window when exit',
+  'auto-resume-all': 'Automatically resume all unfinished tasks',
+  'default-dir': 'Default Download Directory',
```
Comment (agalwood) [inline src/shared/locales/en-US/preferences.js]: Sometimes for layout considerations, you need to simplify the wording. `Default Download Directory` is too long.

File: src/shared/locales/en-US/window.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ export default {
   'reload': 'Reload',
   'close': 'Close',
   'minimize': 'Minimize',
-  'zoom': 'Zoom',
+  'zoom': 'Maximize',
```
Comment (agalwood) [inline src/shared/locales/en-US/window.js]: `Zoom` is the usual name on macOS. It is not only maximized. Please revert this change.

https://electronjs.org/docs/api/menu-item#roles

File: src/shared/locales/en-US/preferences.js (hunk 2 above)
Comment (gl14916) [reply src/shared/locales/en-US/preferences.js]: I updated it to `Default Path`
